hackclub.com.yaml

```
@@ -1683,6 +1683,10 @@ hackaruto:
   - ttl: 600
     type: CNAME
     value: hackaruto.netlify.app.
+hackarest:
+  - ttl: 600
+    type: A
+    value: 86.124.168.69.
```

No trailing dot for IPs
```suggestion
    value: 86.124.168.69
```